packages/c-close-button/src/c-close-button.tsx

```diff
@@ -69,21 +63,18 @@ export const CCloseButton = defineComponent({
 
       const styles = useStyleConfig("CloseButton", themingProps)
 
-      return h(
-        chakra(props.as, {
-          label: "icon-button",
-          __css: {
-            ...baseStyles,
-            ...styles.value,
-          },
-        }),
-        {
-          type: "button",
-          disabled: props.isDisabled,
-          "aria-label": "Close",
-          ...attrs,
-        },
-        slots.default ? slots : () => [h(CCloseIcon)]
+      return (
+        <chakra.button
+          as={props.as}
+          type="button"
+          disabled={props.isDisabled}
+          aria-label="Close"
+          __label="icon-button"
+          __css={{ ...baseStyles, ...styles.value }}
+          {...attrs}
+        >
+          {slots.default ? () => getValidChildren(slots) : () => <CCloseIcon />}
```

Comment:
This ternary will not invoke the render, but instead return the string of the render itself. (See the vite playground result)

Suggest using this instead:
```tsx
{() => (slots.default ? getValidChildren(slots) : <CCloseIcon />)}
```

Reply:
Woops indeed, thanks for pointing that out, will fix it now :)